tictactoe.py

```diff
@@ -0,0 +1,73 @@
+class Tic_Tac_Toe:
+    def __init__ (self):
+        self.board = ['1','2','3','4','5','6','7','8','9']
+        self.options = ['1','2','3','4','5','6','7','8','9']
+        self.num_of_turns = 0 
+
+
+    def display_board(self):
+        print(self.board[0] + ' | ' + self.board[1] + ' | ' + self.board[2])
+        print("------------")
+        print(self.board[3] + ' | ' + self.board[4] + ' | ' + self.board[5])
+        print("------------")
+        print(self.board[6] + ' | ' + self.board[7] + ' | ' + self.board[8])
+
+    def play_game(self): #There must be two players
+        self.display_board()
+        while (self.winner() is not True):
+            self.turns('X')
+            if self.winner() is True:
+                break
+            self.turns('O')
+            self.winner()
+            
+    def turns(self,player_symbol):
+        print('\\n\\n\\n')
+        print("It's " + player_symbol + "'s turn" )
+        position = input('Pick a number displayed on the board ')
+
+        if (position not in self.options):
+            print('')
+            print('Error, number not an option try again')
+            self.display_board()
+            self.turns(player_symbol)
+        else:
+            self.options.remove(position)
+            self.board[int(position)-1] = player_symbol
+            self.num_of_turns += 1
+            self.display_board()
+            if self.num_of_turns == 9:
+                self.winner()
+                print( 'TIE! Rematch?')
+                return
+
+    def winner(self):
+        if self.col_win():
```

Comment:
I like this method,
naming is a bit strange `check_winner()` ?

can also reduce entire method to something like
`return self.col_win() or self.row_win() or self.diag_win()`